apple/internal/run_support.bzl

```diff
@@ -37,7 +37,25 @@ def _register_simulator_executable(ctx, output):
         },
     )
 
+def _register_app_executable(ctx, output):
+    """Registers an action that runs the bundled app.
+
+    Args:
+      ctx: The Skylark context.
+      output: The `File` representing where the executable should be generated.
+    """
+    ctx.actions.expand_template(
+        output = output,
+        is_executable = True,
+        template = ctx.file._macos_runner_template,
+        substitutions = {
+            "%app_name%": ctx.label.name,
+            "%ipa_file%": outputs.archive(ctx).short_path,
```

Comment:
Ipa file probably isn't the right term

Reply:
Yeah I’m not sure what to call this. “app_bundle_path”?

Reply:
That or app_path sounds fine to me

Reply:
Updated!